src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -179,6 +180,21 @@ const CreateImageWizard = () => {
           >
             <OscapStep />
           </WizardStep>
+          <WizardStep
+            name="Content"
+            id="step-content"
+            footer={<CustomWizardFooter disableNext={true} />}
```

Comment:
Since this step will consist of substeps, does it need its own footer? I don't think that the parent step for the target environments has a footer, just the children.

Reply:
Oops, great catch. Removed :sweat_smile: